customization.xml

```diff
@@ -1,72 +1,196 @@
 <application>
   <component name="com.intellij.ide.ui.customization.CustomActionsSchema">
-    <group value="Tool_External Tools_Copyright" is_action="true" action_type="1" position="6">
+    <group value="Tool_External Tools_Copyright" is_action="true" action_type="1" position="0">
+      <path value="root" />
+      <path value="Main menu" />
+      <path value="Tools" />
```

Comment:
@jpsamper2009 It seems we don't need to add these items in "Main menu". \
It looks like CLION adding them there automatically. \
And wise versa, with current changes these items appears in menu twice.

Reply:
@michaelorlov That may be the case... since the buttons don't always show up, I may have tried adding them in more than one place. Will you remove this?

Reply:
@jpsamper2009 Wouldn't you mind if I will push commit addressing this issue and adding new icon on your branch?

Reply:
@MichaelOrlov Go for it.

Reply:
@jpsamper2009 Ok, I've just pushed my changes. Please review them :smile:  \
Please note, that I also removed button for PCLint which is applicable for only one currently selected file. \
Currently `PCLint` button in toolbar will do PCLint check for entire package. \
Although PCLint for one file still available in Menu -> External Tools with the same icon. \
The rational for this is that in most cases we are using PCLint for entire package. And in some rare cases it would be nice to have ability to apply PCLint for currently selected file.

Reply:
@jpsamper2009 ping.

Reply:
@MichaelOrlov Works for me.